modules/core/src/test/java/org/apache/ignite/testframework/junits/logger/GridTestLog4jLoggerSelfTest.java

```diff
@@ -64,28 +71,89 @@ public static void afterTests() {
         assertEquals(defaultRootLevel, LoggerContext.getContext(false).getConfiguration().getRootLogger().getLevel());
     }
 
+    /** */
+    @Test
+    public void testFileConstructor() throws Exception {
+        File xml = GridTestUtils.resolveIgnitePath(LOG_PATH_TEST);
+
+        assert xml != null;
+        assert xml.exists();
```

Comment:
the existence is already tested in `GridTestUtils.resolveIgnitePath`

Reply:
Copied this part of code from Log4j2LoggerSelfTest. Now fixed it in GridTestLog4jLoggerSelfTest and in Log4j2LoggerSelfTest